iidm/iidm-api/src/main/java/com/powsybl/iidm/network/GeneratorAdder.java

```diff
@@ -35,10 +35,12 @@
     GeneratorAdder setVoltageRegulatorOn(boolean voltageRegulatorOn);
 
     /**
-     * Set the regulating terminal, If not set or set to null local terminal is used.
+     * Set the regulating terminal.
      */
     GeneratorAdder setRegulatingTerminal(Terminal regulatingTerminal);
 
+    GeneratorAdder useLocalRegulation(boolean use);
```

Comment:
This should have a default implementation:
```java
default GeneratorAdder useLocalRegulation(boolean use) {
      return this; // does nothing
}
```
Maybe add javadoc too

Reply:
Instead of leaving an empty implementation, I think we should throw an unsupported operation exception, keeping the same strategy we have used when extending interfaces for shunt compensator and static var compensator adders. Do you agree @MioRtia ?